metagraph/src/graph/alignment/aligner_methods.hpp

```diff
@@ -110,62 +105,67 @@ class ExactSeeder : public ExactMapSeeder<NodeType> {
     typedef DBGAlignerConfig::score_t score_t;
 
     ExactSeeder(const DeBruijnGraph &graph, const DBGAlignerConfig &config)
-          : ExactMapSeeder<NodeType>(graph, config) {}
+          : graph_(graph), config_(config) { assert(config_.check_config_scores()); }
 
-    void call_seeds(std::function<void(Seed&&)> callback) const;
+    void call_seeds(std::function<void(Seed&&)> callback) const override;
+
+    const DeBruijnGraph& get_graph() const override { return graph_; }
+    const DBGAlignerConfig& get_config() const override { return config_; }
```

Comment:
same, you probably don't need these getters...

Reply:
I made these protected instead